src/main/webapp/scripts/popular.js

```diff
@@ -1,14 +1,20 @@
+import { getTrendingFromYoutubeApi } from './list-video-service.js';
+
 function loadPopular() {
-  loadApi(function() {
-    getTrendingFromYoutubeApi()
-    .then((response) => {
-      for(item of response.items) {
-        // Ready in a later PR: display the item nicely on the page
-        console.log(item);
-      }
-    })
-    .catch((error) => {
-      updateDom(error.message, 'popular-list-container');
-    });
+  return loadApi().then(() => popular.getTrendingFromYoutubeApi()).then((response) => {
+    for(let item of response.items) {
+      // Ready in a later PR: display the item nicely on the page
+      const containerDOM = document.getElementById('popular-list-container');
+      containerDOM.innerText += item;
+    }
+  })
+  .catch((error) => {
+    updateDom('An error occured with YouTube', 'popular-list-container');
+    console.error(error);
```

Comment:
Nit: Wonder if you can just re-throw the error instead of just an error log.

Reply:
I commented somewhere else but QUnit won't let tests pass if there are uncaught errors, another option is to catch it when qunit calls it but that was kinda weird to me, or alternatively assert that the function threw an error instead of asserting the dom was changed correctly but that was weird to do since its an asynchronous function. I'll leave it as-is for now and add an issue.